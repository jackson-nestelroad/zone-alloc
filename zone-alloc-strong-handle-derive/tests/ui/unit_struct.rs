use zone_alloc_strong_handle_derive::StrongHandle;

#[derive(StrongHandle)]
struct IntegerHandle;

fn main() {}
