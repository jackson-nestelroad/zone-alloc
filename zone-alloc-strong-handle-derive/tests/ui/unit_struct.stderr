error: input must be a tuple struct with unnamed fields
 --> tests/ui/unit_struct.rs:3:10
  |
3 | #[derive(StrongHandle)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `StrongHandle` (in Nightly builds, run with -Z macro-backtrace for more info)
